CMakeLists.txt

```diff
@@ -461,8 +461,13 @@ add_compile_definitions(FOLLY_HAVE_INT128_T=1)
 set_source(folly)
 resolve_dependency(folly)
 
-if(${VELOX_BUILD_TESTING})
-  # Spark qury runner depends on absl, gRPC.
+if(VELOX_BUILD_FUZZERS)
```

Comment:
Is the fuzzer part of testing? Wondering if it makes sense to check both VELOX_BUILD_TESTING and VELOX_BUILD_FUZZERS.

Reply:
My thinking is that fuzzers and unit tests should be independent. 
`VELOX_BUILD_TESTING` should enable units tests.
`VELOX_BUILD_FUZZERS` should enable fuzzers.

Reply:
While we might build fuzzers when we also build testing it does make sense to make them independent, would save us build time in the fuzzer builds and test builds respectivly.